sdcm/utils/common.py

```diff
@@ -1148,38 +1148,53 @@ def filter_gce_instances_by_type(instances):
     return filtered_instances
 
 
-BUILDERS = [
-    {
-        "name": "aws-eu-builder-2",
-        "public_ip": "34.249.55.96",
-        "user": "jenkins",
-        "key_file": os.path.expanduser("~/.ssh/scylla-qa-ec2")
-    },
-    {
-        "name": "aws-eu-builder-3",
-        "public_ip": "34.248.136.231",
-        "user": "jenkins",
-        "key_file": os.path.expanduser("~/.ssh/scylla-qa-ec2")
-    },
-    {
-        "name": "aws-eu-builder-4",
-        "public_ip": "99.81.48.181",
-        "user": "jenkins",
-        "key_file": os.path.expanduser("~/.ssh/scylla-qa-ec2")
-    },
-    {
-        "name": "aws-eu-builder-5",
-        "public_ip": "18.200.171.172",
-        "user": "jenkins",
-        "key_file": os.path.expanduser("~/.ssh/scylla-qa-ec2")
-    },
-    {
-        "name": "aws-eu-builder-6",
-        "public_ip": "99.81.50.63",
-        "user": "jenkins",
-        "key_file": os.path.expanduser("~/.ssh/scylla-qa-ec2")
-    }
-]
+SSH_KEY_DIR = "~/.ssh"
+SSH_KEY_AWS_DEFAULT = "scylla-qa-ec2"
+SSH_KEY_GCE_DEFAULT = "scylla-test"
+
+
+def get_aws_builders():
+    builders = []
+    ssh_key_path = os.path.join(SSH_KEY_DIR, SSH_KEY_AWS_DEFAULT)
+
+    aws_builders = list_instances_aws(tags_dict={
+        "NodeType": "Builder"
+    })
+
+    for aws_builder in aws_builders:
+        builder_name = [tag["Value"] for tag in aws_builder["Tags"] if tag["Key"] == "Name"][0]
+        builders.append({
+            "public_ip": aws_builder["PublicIpAddress"],
+            "name": builder_name,
+            "user": "jenkins",
+            "key_file": os.path.expanduser(ssh_key_path)
+        })
+
+    return builders
+
+
+def get_gce_builders():
+    builders = []
+    ssh_key_path = os.path.join(SSH_KEY_DIR, SSH_KEY_GCE_DEFAULT)
+
+    gce_builders = list_instances_gce(tags_dict={"nodetype": "builder"})
```

Comment:
why `nodetype` is different from AWS `NodeType`?

Reply:
I changed the tags on the Gce builders. Just change in the code to `NodeType`

Reply:
resolved

Reply:
I don't see it is resolved...
![image](https://user-images.githubusercontent.com/5269241/69744241-dc578300-1150-11ea-9a4c-070b97d585ce.png)

Reply:
Fixed